Source/appfat.cpp

```diff
@@ -7,10 +7,12 @@
 #include "../3rdParty/Storm/Source/storm.h"
 #include <config.h>
 
-DEVILUTION_BEGIN_NAMESPACE
+namespace dvl {
+
+namespace {

```

Comment:
I'm confused by this second empty namespace. Is this some sort of feature in C++ that I'm not aware of?

Reply:
This is how you mark things that are private to this translation unit. It allows the compiler to perform aggressively to the contained code since it knows it cannot be called from other parts of the program. It's similar to marking functions as static.

Reply:
Very interesting (if weird... as with a lot of C++ stuff I guess). Thanks for clarifying!